### PR TITLE
Creativity section

### PR DESCRIPTION
**Pull request by InetaVei**
changed icons colors and hover. 
I think it looks ok, but it's not perfect yet.

### CODE REVIEW

File: css/components/creativity.css [hunk 1]
```diff
@@ -48,10 +46,22 @@ section.creativity > .row > .col-xs-12 {
     margin-top: 15px;
     text-align: center;
 }
-#creat-icons {
-  background: url(/images/hero/bg.jpg);
+#cIcons {
```
Comment (belauzas) [inline css/components/creativity.css]: CSS **labai** nerekomenduoju naudoti ID, nes po to bus **labai** sunki tokius selektorius permušti -> keičiam į klasę 👍

File: css/components/creativity.css [hunk 2]
```diff
@@ -48,10 +46,22 @@ section.creativity > .row > .col-xs-12 {
...
+  display: inline;
+  font-size: 90px; 
+  padding: 25px 100px 140px 80px;
+  background-clip: padding-box;
+  -webkit-background-clip: text;
+  color: rgba(0, 0, 0, 0);
+  background-image: linear-gradient(90deg, #12b4c8, #00aeef, #09c778, #01a0f9);
+  opacity: 1;
+}
+#cIcons:hover {
```
Comment (belauzas) [inline css/components/creativity.css]: Same... no ID's -> CSS class 👍

**Review by belauzas — CHANGES_REQUESTED** [review]
Need some fixing 👍

**Comment by InetaVei** [comment]
changed icons ID to class

**Review by belauzas — APPROVED** [review]

**Review by JurateJ — APPROVED** [review]

**Review by Ramazinas — APPROVED** [review]

**Review by Jjurgita — APPROVED** [review]

**Review by EgleJuske — APPROVED** [review]

**Pull request merged**